src/EntityRepository/CustomerRepository.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\EntityRepository;
+
+use App\Entity\Customer;
+use Doctrine\ORM\EntityRepository;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+
+/**
+ * @ORM\Entity
+ */
+class CustomerRepository extends ServiceEntityRepository
```

Comment:
CustomerRepository class should extend EntityRepository because ServiceEntityRepository constructor requires two parameters to be passed, none given. Otherwise, you should use autowiring and configure these arguments in services.yaml file.